velox/vector/ConstantVector.h

```diff
@@ -44,25 +44,7 @@ class ConstantVector final : public SimpleVector<T> {
       velox::memory::MemoryPool* pool,
       size_t length,
       bool isNull,
-      T&& val,
-      const SimpleVectorStats<T>& stats = {},
-      std::optional<ByteCount> representedBytes = std::nullopt,
-      std::optional<ByteCount> storageByteCount = std::nullopt)
-      : ConstantVector(
-            pool,
-            length,
-            isNull,
-            CppToType<T>::create(),
-            std::move(val),
-            stats,
-            representedBytes,
-            storageByteCount) {}
-
-  ConstantVector(
```

Comment:
Maybe update the PR title to say we're also changing ConstantVector?

Reply:
Actually, this is a stacked PR. I'm updating ConstantVector in #4072

Reply:
Ah, I hate that github doesn't properly support them :(